src/com/lilithsthrone/game/character/body/valueEnums/LegConfiguration.java

```diff
@@ -654,6 +654,65 @@ public void setLegsToDemon(GameCharacter character) {
 		public boolean isTailLostOnInitialTF() {
 			return false;
 		}
+	},
+
+	/**
+	 * This LegConfiguration is a 'tauric' configuration for bird races.
+	 */
+	WINGED_BIPED("winged-biped",
+			0,
+			0,
+			false,
+			true,
+			WingSize.THREE_LARGE,
+			false,
+			2,
+			"A configuration in which the character's legs and groin are replaced by the body of the associated animal-morph, with their genitals shifting to be found in a rear-facing cloaca."
+					+ " The most common examples of this are feral wyverns and feral bats, in which the arms are replaced by wings, though they tend to walk on all fours by using their wings like forelegs.",
+			"Above [npc.her] groin, occupying the lower region of [npc.her] humanoid abdomen,",
+			TFModifier.TF_MOD_LEG_CONFIG_WINGED_BIPED,
+			"statusEffects/race/raceBackgroundLegAvian") {
+		@Override
+		public List<GenitalArrangement> getAvailableGenitalConfigurations() {
+			return Util.newArrayListOfValues(
+					GenitalArrangement.NORMAL,
+					GenitalArrangement.CLOACA,
+					GenitalArrangement.CLOACA_BEHIND);
+		}
+		@Override
+		public List<Class<? extends BodyPartInterface>> getFeralParts() {
+			return Util.newArrayListOfValues(Ass.class, Anus.class, BreastCrotch.class, Leg.class, Tail.class, Tentacle.class, Penis.class, Testicle.class, Vagina.class, Clitoris.class);
+		}
+		@Override
+		public List<BodyPartClothingBlock> getBodyPartClothingBlock(GameCharacter character) {
+			if(character.isFeral()) {
+				return Util.newArrayListOfValues(
+							new BodyPartClothingBlock(
+									Util.newArrayListOfValues(
+											InventorySlot.WEAPON_MAIN_1,
+											InventorySlot.WEAPON_MAIN_2,
+											InventorySlot.WEAPON_MAIN_3,
+											InventorySlot.WEAPON_OFFHAND_1,
+											InventorySlot.WEAPON_OFFHAND_2,
+											InventorySlot.WEAPON_OFFHAND_3),
+									character.getLegType().getRace(),
+									"Due to the fact that [npc.nameHasFull] the feral body of [npc.a_legRace], [npc.she] cannot wield regular weapons!",
+									Util.newArrayListOfValues(
+											ItemTag.FITS_ARM_WINGS,
+											ItemTag.FITS_ARM_WINGS_EXCLUSIVE)));
```

Comment:
As they have no wings or paws or similar to 'hold' any weapon, only weapons that are designed to be attached to wings should work here. e. g. some kind of spikes or blades since a strike from arm wings is more, like a [wing buffet](https://classic.wowhead.com/spell=18500/wing-buffet).
The same could be applied to feral avians, but this might be debatable since they don't work well for a wing buffet.
And since not every avian body has wings instead of arms you might wanna use something, like:
```java
if(character.isFeral()) {
	return Util.newArrayListOfValues(
		!character.getArmType().allowsFlight() ? null : new BodyPartClothingBlock(
		// [snipp]
		Util.newArrayListOfValues(
			ItemTag.FITS_ARM_WINGS,
			ItemTag.FITS_ARM_WINGS_EXCLUSIVE)));
}
```